pkg/machine/e2e/config_init_test.go

```diff
@@ -90,6 +90,11 @@ func (i *initMachine) buildCmd(m *machineTestBuilder) []string {
 			if strings.Contains(session.errorToString(), "VM does not exist") {
 				return
 			}
+
+			// The exception below is required for testing custom ignition files.
+			if strings.Contains(session.errorToString(), "failed to remove machines files: unable to find connection named") {
+				return
+			}
```

Comment:
Can you file a new bug for this add a then add a `FIXME: #xxx work around for custom ignition removal` comment here? and replace xxx with the real github issue number